test/functional/tools/collection_discover_dataset_test.xml

```diff
@@ -0,0 +1,31 @@
+<tool id='discover_test' name='discover_test' version='0.1.0' python_template_version='3.5'>
```

Comment:
I don't think Galaxy will run tests on this if you don't add it to https://github.com/galaxyproject/galaxy/blob/dev/test/functional/tools/samples_tool_conf.xml

Reply:
Thanks, added!